ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -37,6 +37,11 @@ gcp_connection::gcp_connection(const std::string &bucket_name, const std::string
     : _gcp_client(google::cloud::storage::Client()), _bucket_name(bucket_name),
       _object_prefix(prefix)
 {
+    bool exists = false;
+
+    bucket_exists(exists);
+    if (!exists)
+        std::cerr << "The specified bucket does not exist." << std::endl;
```

Comment:
I suggest changing the error output to
```
std::cerr << "Bucket '" << bucket_name << "' does not exist." << std::endl;
```
It's odd to have a `bucket_exists` function that doesn't take a bucket name as a parameter. Is there a reason we want to separate that code out into a function?

Reply:
bucket exist is now being checked inside the constructor here: [e806560](https://github.com/wiredtiger/wiredtiger/pull/8737/commits/e806560cfcd70355152938a098de63b3c5071635)